tests/logprob/test_order.py

```diff
@@ -293,3 +294,18 @@ def test_min_max_bernoulli():
     min_logp_fn = pytensor.function([value], pm.logp(pt.min(x), value))
     np.testing.assert_allclose(min_logp_fn(1), np.log(p**n))
     np.testing.assert_allclose(min_logp_fn(0), np.log(1 - p**n))
+
+
+def test_non_measurable_max_grad():
+    # Regression test for https://github.com/pymc-devs/pytensor/issues/711
+    x = pt.random.normal(0, 1, size=(3,))
+    max_x = x.max()
```

Comment:
Is there any value-add to including a test that uses the `axis` kwarg?

Reply:
That looks irrelevant for what caused the original issue